content/docs/Getting started/setting-up-the-database.md

```diff
@@ -154,8 +154,11 @@ To install the database:
 
 After you've installed PostgreSQL, you must create a database.
 
-This assumes your current OS user ($USER) is a db superuser of the same name,
-and that you are able to log in without a password.
+> **Note:** This assumes a PostgreSQL superuser role has been created with the
```

Comment:
I believe these callout sections are done differently in Docsy. See for example the "To make things easier, install a Postgres client:" section.